docs/resources/Auto_Moderation.md

```diff
@@ -0,0 +1,165 @@
+# Auto Moderation
+
+Auto Moderation is a per [guild](#DOCS_RESOURCES_GUILD/) feature which allows the configuration of a set of rules
+to trigger based on some criteria, e.g. a message contains a specific keyword. Certain actions, such as blocking a message,
+can be setup to automatically execute whenever a rule is triggered.
+
+### Auto Moderation Rule Object
+
+###### Auto Moderation Rule Structure
```

Comment:
Shouldn't the structure for this include an `id`? Both the PATCH and DELETE endpoints reference an `id` from the auto moderation rule.

Reply:
Yes! Sorry, I forgot it. Done!